pyiron_gui/project/project_browser.py

```diff
@@ -23,6 +29,207 @@
 __date__ = "Feb 02, 2021"
 
 
+class PyironWrapper:
+
+    """Simple wrapper for pyiron objects which extends for basic pyiron functionality (list_nodes ...)"""
+
+    _with_self_representation = {
+        "structure": 'pyiron_atomistics.atomistics.structure.atoms.Atoms',
```

Comment:
I always get a bit uncomfortable with these string references to objects because they're at a higher risk of getting out-of-sync when we refactor the code. I know for the `JOB_CLASS_DICT` I didn't see a way around it, but here it looks like you just use them for an instance check. Would it be possible to replace this with `from pyiron_atomistics import Atoms ... 'structure': Atoms` or even `Atoms.__name__` or something if you still want to use the string?

I'm not 100% against these string uses, I just want to avoid them wherever it won't overly complicate the implementation.

Reply:
So far I just use only `pyiron_base` features and would import `pyiron_atomistics` only for `isinstance` checks. However, probably `pyiron_gui` will import and rely on all the `pyiron_*` repos at some point so I will start using the real classes :)